### PR TITLE
Modify Logging Exporter test based on download URL https://github.com/oracle/weblogic-logging-exporter/releases/latest

### PR DESCRIPTION
**Pull request by hzhao-github**
Always use latest version of WebLogic Logging Exporter

Jenkin Run(s) 

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/8975/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/8974/

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9080

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java [hunk 1]
```diff
@@ -105,6 +105,7 @@ public static InstallParams defaultInstallWitParams() {
    * @return an InstallParams instance
    */
   public static InstallParams defaultInstallWleParams() {
+    System.out.println("=============================WLE_DOWNLOAD_URL: " + WLE_DOWNLOAD_URL);
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java:108]: Can you use logger.info("Resolved WLE_DOWNLOAD_URL is {0}", WLE_DOWNLOAD_URL);

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java]: this is a debugger. I removed it.

**Comment by anpanigr** [comment]
Can u run the tests against the alpha release  (Release 2.0-alpha-1)
https://github.com/oracle/weblogic-logging-exporter/releases

**Review by vanajamukkara — APPROVED** [review]
LG

**Review by anpanigr — APPROVED** [review]

**Pull request merged**